region/norfair/crocomire/Grapple Tutorial Room 3.json

```diff
@@ -111,13 +112,81 @@
         }
       }
     },
+    {
+      "link": [1, 1],
+      "name": "Leave With Runway - Frozen Gamet",
+      "requires": [
+        "h_canNavigateUnderwater",
+        {"or": [
+          "h_canTrickyFrozenEnemyRunway",
+          {"and": [
+            "h_canFrozenEnemyRunway",
+            "Morph"
+          ]}
+        ]}
+      ],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 6,
+          "openEnd": 0
+        }
+      },
+      "note": [
+        "Lure the Geemers up out of the water. It may be helpful to freeze them on the ascent to prevent them from separating and moving horizontally.",
+        "At the top, freeze a Gamet while there is a half-tile gap between it and the runway in order to extend it as much as possible.",
+        "Positioning is much easier with Morph; simply morph on the runway before they start separating.",
+        "Freezing one or two Gamets after they separate but before they start moving can help with dodging and avoiding damage."
+      ]
```

Comment:
```suggestion
      "note": [
        "Lure the Gamets up out of the water. It may be helpful to freeze them on the ascent to prevent them from separating and moving horizontally.",
        "Positioning is much easier with Morph; simply morph on the runway before they start separating.",
        "Freeze a Gamet while there is a half-tile gap between it and the runway in order to extend it as much as possible."
      ]
```